frontend/public/components/storage/create-pvc.tsx

```diff
@@ -110,6 +113,7 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
         storageClass: _.get(storageClass, 'metadata.name'),
         allowedAccessModes: modes,
         accessMode: 'ReadWriteOnce',
+        provisioner: storageClass.provisioner,
```

Comment:
I guess this is OK, but I feel like it would be better to add the storage class itself to state if we need multiple fields from it. If we're backporting the fix, I am OK making the smaller change.

Reply:
You need to check for a undefined `storageClass` or this could be a runtime error.